source/Nevermore.Tests/RelationalStoreFixture.cs

```diff
@@ -25,7 +25,7 @@ public void ShouldSetDefaultConnectionStringOptions()
         [Test]
         public void ShouldNotOverrideExplicitConnectionStringOptions()
         {
-            var store = new RelationalStore("Server=(local);Connection Timeout=123;ConnectRetryCount=123;ConnectRetryInterval=59;", "Nevermore test", null, null, null, null, 20);
+            var store = new RelationalStore("Server=(local);Connection Timeout=123;ConnectRetryCount=123;ConnectRetryInterval=59;", "Nevermore test", null, new RelationalStoreConfiguration(), null, 20);
```

Comment:
So much crap needed for these constructors! At least `RelationalStoreConfiguration` removes one parameter.